components/Sections/NewHeader/__tests__/__snapshots__/index.test.tsx.snap

```diff
@@ -0,0 +1,437 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Is this snapshot file actually needed? Do we still have any snapshot on the regular test file?

Reply:
We had still, but I removed them, was no real value in them